p11-kit/list-objects.c

```diff
@@ -274,21 +275,22 @@ print_object (p11_list_printer *printer,
 
 	p11_kit_iter_get_attributes (iter, attrs, n_attrs);
 	for (i = 0; i < n_attrs; ++i) {
+		p11_buffer buffer;
```

Comment:
I would prefer to have this variable declaration at the start of the function with the rest of the declarations.

Reply:
I don't think that is beneficial; there are many other projects suggest the opposite (i.e., limit the variable scope, [example](https://github.com/libreswan/libreswan/blob/main/docs/ProgrammingConventions.txt#L80))

Reply:
I was violating this principle all the time then. Good to know